resources/js/Jetstream/Checkbox.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
This is a new component, it should go in `js/Components` and not `js/Jetstream`

Reply:
No, its from jetstream v2, as mentioned in the commit. should stay with the other original vendor components.